pom.xml

```diff
@@ -61,4 +61,31 @@
         </license>
     </licenses>
 
+    <build>
+        <pluginManagement>
+            <plugins>
+               <plugin>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
+                    <version>1.27.0</version>
+                    <executions>
+                        <execution>
+                            <id>spotless-check</id>
+                            <goals>
+                                <goal>check</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <java>
+                            <eclipse>
+                                <file>https://raw.githubusercontent.com/eclipse-ee4j/ee4j/${project.parent.parent.version}/codestyle/ee4j-eclipse-formatting.xml</file>
```

Comment:
It was the worst I could do. In local parent refer to parent.parent because it works in child :cry:
I'll move configuration down to api.